setup.cfg

```diff
@@ -34,3 +34,22 @@ ignore_missing_imports = True
 [flake8]
 max-line-length = 200
 exclude = venv/*,var/*
+
+# E203 - White space before ':', conflicting with black
+# E712 - comparison to False should be 'if cond is False:' or 'if not cond:', doesn't apply to our code
+# E731 - do not assign a lambda expression, use a def, doesn't apply to our code
+
+# F401 - module imported but not used
+
+# W503 - line break before binary operator, conflicting with black
+# W605 - invalid escape sequence, modifying will cause test failures
+
+ignore = E203,E712,E731,W503
+per-file-ignores =
+  #  JsonApiClient used in comments, black doesn't inline comments in that line
```

Comment:
Can you clarify what this means?

Reply:
The inline above comments was split into multiple lines then purpose of suppressing flake8 errors isn't served. So I do it here again. But would like to leave the inline comments there so other people can be aware of.

Reply:
Can you share the links to the lines that are problematic? I will take a look.

I think it's better to ignore a line or two within the file instead of ignoring a rule for the whole file.

Reply:
The module is imported in this [line](https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/bundle_util.py#L5), and it's used [here](https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/bundle_util.py#L56).

Reply:
The problem is if I add the inline comments to ignore it, then the black will split my comments to the following lines and that will make it won't work.

Reply:
Yeah we shouldn't ignore a rule for an entire file for just that one line.